lighthouse-core/gather/computed/speedline.js

```diff
@@ -30,14 +30,24 @@ class Speedline extends ComputedArtifact {
       return speedline(traceEvents, {
         timeOrigin: navStart,
         fastMode: true,
-        include: 'perceptualSpeedIndex',
+        include: 'speedIndex',
       });
     }).catch(err => {
       if (/No screenshots found in trace/.test(err.message)) {
         throw new LHError(LHError.errors.NO_SCREENSHOTS);
       }
 
       throw err;
+    }).then(speedline => {
+      if (speedline.frames.length === 0) {
```

Comment:
this was done just in the audit, but it makes sense to be doing this everywhere speedline is needed, moved the test as well